pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -639,4 +639,92 @@ rec {
     };
   };
 
+  ivyde = buildEclipsePlugin rec {
+    name = "ivyde-${version}";
```

Comment:
Does buildEclipsePlugin not accept pname + version?

Reply:
This was my very first pull request for nixos and I simply copied from existing plugins. All existing plugins use that way for the name, so I believe it's the only way to do so.